File: server/router/admin/gost_client_forward.go
package admin

import (
	"github.com/gin-gonic/gin"
	"server/controller/admin/gost_client_forward"
	"server/global"
	"server/router/middleware"
)

func InitGostClientForward(group *gin.RouterGroup) {
	g := group.Group("gost/client/forward", middleware.Auth(global.Jwt), middleware.AuthAdmin())
	g.POST("create", gost_client_forward.Create)
	g.POST("config", gost_client_forward.Config)
	g.POST("delete", gost_client_forward.Delete)
	g.POST("page", gost_client_forward.Page)
}

File: web/src/api/admin/gost_client_forward.js
import request from "../../request/index.js";

const baseUrl = '/v1/admin/gost/client/forward'

export const apiAdminGostClientForwardList = () => {
    return request.request({
        url: `${baseUrl}/list`,
        method: 'POST',
    })
}

export const apiAdminGostClientForwardPage = (data) => {
    return request.request({
        url: `${baseUrl}/page`,
        method: 'POST',
        data
    })
}

export const apiAdminGostClientForwardCreate = (data) => {
    return request.request({
        url: `${baseUrl}/create`,
        method: 'POST',
        data
    })
}

export const apiAdminGostClientForwardConfig = (data) => {
    return request.request({
        url: `${baseUrl}/config`,
        method: 'POST',
        data
    })
}

export const apiAdminGostClientForwardDelete = (data) => {
    return request.request({
        url: `${baseUrl}/delete`,
        method: 'POST',
        data
    })
}